### PR TITLE
cli/atlas: product review

### PR DESCRIPTION
**Pull request by yonidavidson**
- includes all changes in review (doesn't include postgress errors found today).

### CODE REVIEW

File: cmd/atlas/mux.go [hunk 1]
```diff
@@ -71,7 +71,7 @@ func (u *Mux) OpenAtlas(dsn string) (*Driver, error) {
 	}
 	p, ok := u.providers[key]
 	if !ok {
-		return nil, fmt.Errorf("could not find provider, %s", err)
+		return nil, fmt.Errorf("could not find provider, %s", key)
```
Comment (a8m) [inline cmd/atlas/mux.go:74]:
```suggestion
		return nil, fmt.Errorf("could not find provider: %s", key)
```

File: cmd/atlas/schema.go [hunk 2]
```diff
@@ -7,8 +7,8 @@ import (
 // schemaCmd represents the schema command
 var schemaCmd = &cobra.Command{
 	Use:   "schema",
-	Short: "Work with atlas schemas.",
-	Long:  `Provides ability to interact with schema and data source.`,
+	Short: "Work with atlas schemas",
+	Long:  `Interact with the schema and data source`,
```
Comment (a8m) [inline cmd/atlas/schema.go:11]:
```suggestion
	Long:  "Interact with the schema and data source",
```

Why the inconsistency between the 2 strings?

**Review by a8m — APPROVED** [review]

File: cmd/atlas/schema.go (hunk 2 above)
Comment (yonidavidson) [reply cmd/atlas/schema.go:11]: short is when it's a sub command.
long is when you are -h on a specific command.

**Comment by yonidavidson** [comment]
@a8m  - short command - is the text you see when you are looking at a sub command  (for example inspect under help).
lonh -is the help you see when quering a specific command for example "atlas schema inspect -h

**Pull request merged**